src/SerialPortComms.cs

```diff
@@ -1,54 +1,50 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.IO.Ports;
-using System.Linq;
-using System.Text;
 using System.Threading;
-using System.Threading.Tasks;
 
 namespace COMSpike
 {
     public class SerialPortComms
     {
-        private SerialPort port;
+        private readonly SerialPort _port;
```

Comment:
Typically you'd use the target naming conventions for a PR... but this is fine 🤣

Reply:
I didn't follow that, sorry.